privacy-snapshot.json

```diff
@@ -50,5 +49,7 @@
   "lattice.gridplus.io",
   "unresponsive-rpc.test",
   "authentication.api.cx.metamask.io",
-  "oidc.api.cx.metamask.io"
+  "oidc.api.cx.metamask.io",
+  "price.api.cx.metamask.io",
+  "token.api.cx.metamask.io"
```

Comment:
Given the inclusion of `"token.api.cx.metamask.io"` should `"token-api.metaswap.codefi.network",` be removed?

Reply:
I tried to remove it, but there was still a usage somewhere.  I will tackle it separately from this upgrade.

Reply:
Looks like it's becuase this comes from `@metamask/controller-utils`, and sicne this PR introduces the new version but still using the old v9, it will hit a different endpoint dependending on which of the two versions is used.

Reply:
@danjm there's this PR which is currently blocked by this one that removes old metaswap, metafi urls (https://github.com/MetaMask/metamask-extension/pull/24698)